app/views/users/new.html.erb

```diff
@@ -0,0 +1,57 @@
+
+<%= form_for @user do |f| %>
+  <% if @user.errors.any? %>
+    <ul>
+      <% @user.errors.each do |error, msg| %>
+        <li><%= "#{error} - #{msg}" %></li>
+      <% end %>
+    </ul>
+  <% end %>
+
+  <p>
+  <%= f.label :username %><br/>
+  <%= f.text_field :username %>
+  </p>
+  <p>
+  <%= f.label :email %><br/>
+  <%= f.email_field :email, :placeholder => "example@example.com" %>
+  </p>
+  <p>
+  <%= f.label :password %><br/>
+  <%= f.password_field :password %>
+  </p>
+  <%= f.submit %>
+ <% end %>
+
+<% if false %>
+<%= form_tag('/users', enforce_utf8: true) do %>
+  <%= label_tag 'user_name', 'Username:' %>
+  <%= text_field_tag 'user_name'%>
+  
+  <%= label_tag 'user_email', 'Email:'%>
+  <%= email_field_tag 'user_email'%>  <br>
+
+  <%= label_tag 'user_password', 'Password:'%>
+  <%= password_field_tag 'user_password'%>
+
+  <%= submit_tag "Save new user"%>
+
+ <% end %>
```

Comment:
This form is returning `param is missing or the value is empty: user `

Reply:
That's because that form has  different params required than that of the form that is active. So that User.new(params) in the Users controller for the new method would have to change relative to that form... But since that form is inactive didn't think it mattered.